app/both/collections/residents.js

```diff
@@ -11,59 +11,58 @@ var ResidentsSchema = new SimpleSchema({
     type: String,
     autoform: {
       readonly() {
-        return !isUserAdmin() ? !isEdit(this) : false
-      }
+        return !isUserAdmin() ? !isEdit(this) : false;
+      },
     },
     custom() {
       return !isUserAdmin()
         ? isEdit(this)
-          ? "notAllowed"
+          ? 'notAllowed'
           : undefined
         : undefined;
-    }
+    },
   },
   lastInitial: {
     type: String,
     max: 1,
     autoform: {
       readonly() {
-        return !isUserAdmin() ? !isEdit(this) : false
-      }
+        return !isUserAdmin() ? !isEdit(this) : false;
+      },
     },
     custom() {
       return !isUserAdmin()
         ? isEdit(this)
-          ? "notAllowed"
+          ? 'notAllowed'
```

Comment:
Again, this type of code is difficult to read. Try to improve readability, at the expense of having a couple more lines of code.